### ISSUE TITLE
Q: preventing sbt console on start

### ISSUE THREAD

**Issue by gaydenko** [dcaoyuan/nbscala#139]
There are use cases when it can be handy to avoid sbt console automatic starting during IDE startup. Is it possible to prevent?

**Comment by dcaoyuan** [comment]
It won't start after the .classpath_nb was created. Or, you can run "sbt netbeans" outside NetBeans before open NetBeans, thus, NetBeans will not try to open console to run "sbt netbeans" by itself.

**Comment by gaydenko** [comment]
I have this structure:
```txt
appRoot
    subprj1
    ...
    subprjN
    root
    project
```
Here `root` is an sbt project aggregating `subprj1`..`subprjN`, and `project` contains `Build.scala` definition. So I have started sbt in `appRoot` dir, and `netbeans` command created `.classpath_nb` in subprojects' and `root` directories.

On the other hand the IDE sees `appRoot` as scala project also. This project isn't needed to code, but, being opened, this `appRoot` project includes `project` directory into git-related operation (and that is handy - to have the only project for all vcs operations). And this is the project which results in starting sbt console on every IDE (re)start.

At the moment I'm trying this workaround: I have created `appRoot/.classpath_nb` file with empty `classpath` element and opened this project. After IDE restarting sbt console wasn't activated. Will see further. So, closing the issue for now.
